Constitution.md

```diff
@@ -97,32 +101,102 @@ The Terms of Office for all Executive Positions at CSSA will last for at most on
 
 ## Article 2b: Executive Officers and Duties
 
-### President(s)
+
+### Co-Presidents
```

Comment:
This seems very disingenuous. Was this approved officially during the 2022-2023 council? It kind of felt like this was allowed through a loophole. We also don't want to allow this in the future. I don't also like that we are removing presidents for co-presidents.